src/current/v24.1/cockroachdb-feature-availability.md

```diff
@@ -31,54 +31,60 @@ Any feature made available in a phase prior to GA is provided without any warran
 
 ## Features in preview
 
 {{site.data.alerts.callout_info}}
 **The following features are in preview** and are subject to change. To share feedback and/or issues, contact [Support](https://support.cockroachlabs.com/hc).
 {{site.data.alerts.end}}
 
-### `READ COMMITTED` isolation level
+### CockroachDB Cloud Folders
 
-[`READ COMMITTED`]({% link {{ page.version.version }}/read-committed.md %}) is a weaker transaction isolation level than `SERIALIZABLE` and is the default isolation level in databases such as PostgreSQL, Oracle, and SQL Server. `READ COMMITTED` isolation allows writes to interleave without aborting transactions and prevents writes from blocking reads, thus minimizing query latency and retries caused by read/write contention.
+[Organizing CockroachDB Cloud clusters using folders](https://www.cockroachlabs.com/docs/cockroachcloud/folders) is in preview. Folders allow you to organize and manage access to your clusters according to your organization's requirements. For example, you can create top-level folders for ea
+ch business unit in your organization, and within those folders, organize clusters by geographic loc
+ation and then by  level of maturity, such as production, staging, and testing. To express interest
+and try it out, contact [Support](https://support.cockroachlabs.com/hc).
 
-### `cockroach` commands
+### Custom Metrics Chart page for CockroachDB {{ site.data.products.cloud }} clusters
 
-The table below lists the [`cockroach` commands]({% link {{ page.version.version }}/cockroach-commands.md %}) available in preview in CockroachDB.
+The [**Custom Metrics Chart** page]({% link cockroachcloud/custom-metrics-chart-page.md %}) for CockroachDB {{ site.data.products.cloud }} clusters allows you to create custom charts showing the time series data for an available metric or combination of metrics.
 
-Command                                     | Description
---------------------------------------------+-------------
-[`cockroach demo`]({% link {{ page.version.version }}/cockroach-demo.md %})     | Start a temporary, in-memory CockroachDB cluster, and open an interactive SQL shell to it.
-[`cockroach sqlfmt`]({% link {{ page.version.version }}/cockroach-sqlfmt.md %}) | Reformat SQL queries for enhanced clarity.
+### Export metrics from CockroachDB {{ site.data.products.dedicated }} clusters
+[Exporting metrics from CockroachDB {{ site.data.products.dedicated }}](https://www.cockroachlabs.com/docs/cockroachcloud/export-metrics) to [AWS CloudWatch](https://aws.amazon.com/cloudwatch/) or [Datadog](https://www.datadoghq.com/) using the [Cloud API](https://www.cockroachlabs.com/docs/cockroachcloud/cloud-api) is in preview. Once the export is configured, metrics will flow from all nodes in all regions of your CockroachDB {{ site.data.products.dedicated }} cluster to your chosen cloud metrics sink.
```

Comment:
this link and the others that use the full cloud docs URLs can be formatted as `{% link cockroachcloud/export-metrics.md %}`  

among other things it will mean that these links point to the preview build instead of the live site